youtube_dl/extractor/prosiebensat1.py

```diff
@@ -133,7 +133,7 @@ class ProSiebenSat1IE(ProSiebenSat1BaseIE):
                             (?:
                                 prosieben(?:maxx)?|sixx|sat1(?:gold)?|kabeleins(?:doku)?|the-voice-of-germany|7tv|advopedia
                             )\.(?:de|at|ch)|
-                            ran\.de|fem\.com|advopedia\.de
+                            ran\.de|fem\.com|advopedia\.de|galileo.tv/video
```

Comment:
Unescaped dot.

Reply:
That happens when you make changes just before commiting and don't test... Sorry! -_-